apps/web/src/services/cost-explorer/composables/use-cost-report-data-analyze-query.ts

```diff
@@ -0,0 +1,52 @@
+import type { ComputedRef } from 'vue';
+import { computed } from 'vue';
+
+import { isEmpty } from 'lodash';
+
+import type { AnalyzeQuery } from '@cloudforet/core-lib/space-connector/type';
+
+import { useCostReportDataApi } from '@/api-clients/cost-analysis/cost-report-data/composables/use-cost-report-data-api';
+import { useServiceQueryKey } from '@/query/core/query-key/use-service-query-key';
+import { useScopedQuery } from '@/query/service-query/use-scoped-query';
+
+
+export const useCostReportDataAnalyzeQuery = ({
+    cost_report_config_id,
+    is_confirmed,
+    query,
+}: {
+    cost_report_config_id?: ComputedRef<string>;
+    is_confirmed?: boolean;
+    query: ComputedRef<AnalyzeQuery | null>;
+}, isUserScope = false) => {
```

Comment:
Suggestion: How about using name `isCostReportPage` than `isUserScope`?
I think, name based on scenario context will easier to understand code for maintenance.
(unless it's a code that will be recycled in many contexts)

Reply:
+ How about putting `isUserScope` in previous factor? 
```
{
    cost_report_config_id?: ComputedRef<string>;
    is_confirmed?: boolean;
    query: ComputedRef<AnalyzeQuery | null>;
    isUserScope: boolean,
}
```